doc/whats_new/v0.21.rst

```diff
@@ -351,6 +353,17 @@ Support for Python 3.4 and below has been officially dropped.
 - |Feature| Allow different dtypes (such as float32) in
   :class:`isotonic.IsotonicRegression` :issue:`8769` by :user:`Vlad Niculae <vene>`
 
+:mod:`sklearn.svm`
+....................
+- |Fix| Fixed an issue in :func:`svm.SVC.decision_function` which would give
+  different results when applied to a subset due to local scaling method in
+  :func:`utils.multiclass._ovr_decision_function`. :pr:`10440` by :user:`Jonathan Ohayon <Johayon>`
+
+:mod:`sklearn.multiclass`
+....................
```

Comment:
not enough ...

Reply:
should be fine this time.

Reply:
no. You should have as many "." as number of chars in the header title

Reply:
sorry, I did not know about that.